### DISCUSSION TITLE
Can "Unchanged translation" check hits be removed automatically?

### DISCUSSION THREAD

**Discussion by kdambekalns** [WeblateOrg/weblate#6791] [Q&A]
I have a ton of translations that are picked up by the "Unchanged translation" check, and rightly so. Is there a way to remove those? I can't tell right now, whether I am to trust the statistics of translation progress on the languages overview page…

**Comment by nijel** [comment]
You can ignore any check at component level, for [Unchanged translation](https://docs.weblate.org/en/latest/user/checks.html#unchanged-translation) you can do that by adding `ignore-same` flag.

**Reply by kdambekalns** [reply]
> You can ignore any check at component level

Ah, sorry for being unclear… I don't want to remove the reported failures, but the actual unchanged translations in my XLIFF files. "Make them missing translations", so to speak.

**Reply by nijel** [reply]
The only option for this is to script it using the API.
